### PR TITLE
Switch to github.com/golang-jwt/jwt and update gin

### PR DESCRIPTION
**Pull request by alexeykazakov**
Also see https://github.com/codeready-toolchain/toolchain-common/pull/203

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -39,4 +36,6 @@ require (
 	sigs.k8s.io/controller-runtime v0.8.3
 )
 
+replace github.com/codeready-toolchain/toolchain-common => github.com/alexeykazakov/toolchain-common v0.0.0-20210802190237-4f50f37bca4e
```
Comment (MatousJobanek) [inline go.mod:39]: reminder

**Review by MatousJobanek — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (alexeykazakov) [reply go.mod]: Updated

**Comment by alexeykazakov** [comment]
PodIdler test is failing. Which should not be related to this change at all. Looks like a flaky test. Re-trying.

**Comment by alexeykazakov** [comment]
this time the member operator didn't start for some reason

**Pull request merged**